src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -3161,6 +3176,14 @@ bool SvgLoader::header()
         }
 
         preserveAspect = loaderData.doc->node.doc.preserveAspect;
+
+        if ((loaderData.doc->node.doc.bboxFlag & BBoxFlag::All) == BBoxFlag::All ||
+            (loaderData.doc->node.doc.bboxFlag & BBoxFlag::Viewbox) == BBoxFlag::Viewbox ||
+            (loaderData.doc->node.doc.bboxFlag & BBoxFlag::WidthHeight) == BBoxFlag::WidthHeight) {
+            if (vw <= 0 || vh <= 0)
+                return false;
```

Comment:
The bool return of header() determines whether the svg image load failed. (valid, invalid)
If vw and vh are 0, it will be invisible, but I don't think it's a load failure. What do you think?
If you don't necessarily have to return false, how about printing a LOG?

Reply:
from [specs ](https://www.w3.org/TR/SVG2/coords.html#ViewBoxAttribute) regarding the viewBox attrib: "A negative value for <width> or <height> is an error and invalidates the ‘viewBox’ attribute. A value of zero disables rendering of the element."
so for values '<0' 'return false' is correct. 
I've included the '=0' case, to stop calling all of the following functions, but you are right, I'll change it

Reply:
I was wrong, negative values invalidate the 'viewbox' attrib and not the whole svg